embedded-graphics/src/style/mod.rs

```diff
@@ -28,9 +28,9 @@ pub use text_style::{TextStyle, TextStyleBuilder};
 macro_rules! primitive_style {
     ($($style_key:ident = $style_value:expr ),* $(,)?) => {{
         #[allow(unused_mut)]
-        let mut builder = $crate::style::PrimitiveStyleBuilder::new();
+        let  builder = $crate::style::PrimitiveStyleBuilder::new();
```

Comment:
```suggestion
        let builder = $crate::style::PrimitiveStyleBuilder::new();
```

Reply:
Maybe we should try to run `rustfmt` nightly with the appropriate options to format the macros.

Reply:
I turned on `format_macro_matchers` and `format_macro_bodies` in the nightly config but it didn't catch this issue specifically. I've left them in because they'll be handy later anyway. I fixed the double space manually.